e2e/utils.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) 2025-present, NDLA.
+ *
+ * This source code is licensed under the GPLv3 license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ */
+
+import { TestInfo } from "@playwright/test";
+import { readFile, writeFile } from "fs/promises";
+
+const MOCK_DIR = "e2e/apiMocks/";
+const TEST_IMAGE_REGEX = "https://api.test.ndla.no/(?!image-api/raw.*).*";
+const LOCALHOST_GRAPHQL_REGEX = "http://localhost:4000/graphql-api/graphql";
+
+export const API_REGEX = new RegExp(`^(${TEST_IMAGE_REGEX}|${LOCALHOST_GRAPHQL_REGEX})$`);
+
+export const removeSensitiveDataFromHar = async (fileName: string) => {
```

Comment:
Her har du mista kommentaren om at denne ikkje fjerner sensitive data men kun skriver har-fila til ei linje.
Er det for å spare plass vi ikkje lagrer denne formatert? Unngå masse linjeskift?